src/server/route.go

```diff
@@ -50,7 +50,6 @@ func registerRoutes() {
 	beego.Router("/api/internal/syncquota", &api.InternalAPI{}, "post:SyncQuota")
 
 	beego.Router("/service/notifications/jobs/webhook/:id([0-9]+)", &jobs.Handler{}, "post:HandleNotificationJob")
-	router.NewRoute().Method(http.MethodPost).Path("/service/notifications/jobs/adminjob/:id([0-9]+)").Handler(handler.NewJobStatusHandler())         // legacy job status hook endpoint for adminjob
```

Comment:
It is possible that there are still some legacy admin jobs running inside Jobservice, so we should not remove these routes?